templates/docs/patterns/hero/index.md

```diff
@@ -17,12 +17,103 @@ The hero pattern is composed of the following elements:
 
 | Element              | Description                                                              |
 | -------------------- | ------------------------------------------------------------------------ |
-| Title (**required**) | <code>h1</code> title text                                               |
-| Subtitle             | <code>h2</code> subtitle text                                            |
-| Description          | <code>p</code> description text                                          |
+| Title (**required**) | Title text (to be placed in `h1` heading)                                |
+| Subtitle             | Subtitle text (using `h2` style)                                         |
+| Description          | Description text (one or more paragraphs)                                |
 | Call to action block | [Call to action block](/docs/patterns/cta-block) beneath the description |
 | Image                | Main hero visual                                                         |
 
+## Jinja Macro
+
+### Parameters
+
+<div style="overflow: auto;">
+  <table style="overflow: visible; width: auto;">
+    <thead>
+      <tr>
+        <th style="width: 20%;">Name</th>
+        <th style="width: 15%;">Is Required</th>
+        <th style="width: 20%;">Type</th>
+        <th style="width: 15%;">Default</th>
+        <th style="width: 30%;">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>layout</code></td>
+        <td>Yes</td>
+        <td>
+          One of:<br>
```

Comment:
```suggestion
          One of:
```

Reply:
I know removing line breaks messes up readability a bit, but having this very tall list of options (with one of them wrapping) seems to blow up the table row too much. Overall, it's just for the sake of reference.

Reply:
Done